packages/document/main-doc/docs/en/guides/basic-features/proxy.mdx

```diff
@@ -40,7 +39,7 @@ import GlobalProxy from "@site-docs-en/components/global-proxy"
 
 ## BFF Proxy
 
-By configuring [`bff.proxy`](/configure/app/bff/proxy), you can proxy BFF API requests to specified services. Unlike other proxy above, it can also be used in the production environment:
+By configuring [`bff.proxy`](/configure/app/bff/proxy), BFF API requests can be proxied to a specified service. Unlike other proxy above, it can also be used in production environment:
```

Comment:
```suggestion
By configuring [`bff.proxy`](/configure/app/bff/proxy), BFF API requests can be proxied to a specified service. Unlike the other proxy methods above, it can be used in the production environment:
```